src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -83,7 +83,7 @@ const Dashboard = ({
             totalDisabilityRating={props.totalDisabilityRating}
           />
         )}
-        <div className="vads-l-grid-container">
+        <div className="vads-l-grid-container medium-screen:vads-u-padding--2 small-desktop-screen:vads-u-padding--0">
```

Comment:
the grid container adds a lot of horizontal padding that we don't want at large screen sizes.